Common.css

```diff
@@ -1,25 +1,25 @@
-@import "//bits.wikimedia.org/static-current/skins/MonoBook/main.css";
-@import "//bits.wikimedia.org/static-current/resources/src/mediawiki.legacy/shared.css";
+@import "//en.wikipedia.org/w/skins/MonoBook/main.css";
+@import "//en.wikipedia.org/w/resources/src/mediawiki.legacy/shared.css";
```

Comment:
JFTR: These `bits.wikimedia.org` had been done by someone else in the working directory (you?), I just committed them.